Library/Formula/log4cxx.rb

```diff
@@ -37,25 +39,137 @@ def install
   end
 end
 __END__
+diff --git a/src/main/cpp/locationinfo.cpp b/src/main/cpp/locationinfo.cpp
+index e76ea29..2e1950c 100644
+--- a/src/main/cpp/locationinfo.cpp
++++ b/src/main/cpp/locationinfo.cpp
```

Comment:
You could try sending a patch to the mailing list if there’s one or to the author, just in case. Last month I got a pull-request accepted on a project that was dead since 2011.